presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -253,7 +250,7 @@ public ConnectorSplitSource getSplits(JdbcTableLayoutHandle layoutHandle)
     public Connection getConnection(JdbcSplit split)
             throws SQLException
     {
-        Connection connection = driver.connect(split.getConnectionUrl(), toProperties(split.getConnectionProperties()));
+        Connection connection = driver.connect(connectionUrl, connectionProperties);
```

Comment:
Dear Reviewer,
although this makes `split` method param unused now, I thought that leaving it here would make it easier to subclass the base-jdbc connector connection-related information needs to be passed within a split.

Reply:
This is definitely needed by custom connectors.